ingest/src/test/java/com/connexta/ingest/IngestApplicationIntegrationTest.java

```diff
@@ -42,6 +52,32 @@ public void beforeEach() {
   @Test
   public void testContextLoads() {}
 
+  @Test
+  public void testSuccessfulRequest() throws Exception {
```

Comment:
Since this just tests if we called the transform service, should it be called `test SuccessfulTransformRequest`?

Reply:
Yeah that's probably more accurate.